index.html

```diff
@@ -1642,192 +1662,172 @@ <h2>Architectural Aspects of a Web Thing</h2>
+                A hypermedia control is the machine-understandable serialization of <emph>how</emph> to activate an affordance.
+                Hypermedia controls usually originate from a Web server and are discovered in-band while a Web client is interacting with the server.
+                This way, Web servers can drive clients through Web applications dynamically,
+                by taking their current state and other factors such as authorization into account.
+                This is opposed to out-of-band interface descriptions that need to be preinstalled or hardcoded into clients
+                (e.g., RPC, WS-* Web services, HTTP services with fixed URI-method-response definitions).
+            </p>
+            <p>
+                W3C WoT makes use of two kinds of hypermedia controls:
+                Web links [[!RFC8288]] as the well-established control to navigate the Web,
+                and Web forms as a more powerful control to enable any kind of operation.
+                The same concept is used in the Constrained RESTful Application Language (CoRAL) defined by the IETF [[?CoRAL]].
+            </p>
             <section id="sec-hypermedia-links">
                 <h3>Links</h3>
                 <p>
-                    Web links enable clients to change the current
-                    context (e.g., the set of resource representations
-                    currently rendered in the Web browser) or to include
-                    further resources in the current context, depending
-                    on the relation between context and link target.
-                    Clients do so by <i>dereferencing</i> the target
-                    URI, that is, fetching the resource representation.
+                    Links enable <a>Consumers</a> (or Web clients in the broader sense) to change the current context
+                    (cf. the set of resource representations currently rendered in the Web browser)
+                    or to include additional resources into the current context,
+                    depending on the relation between context and link target.
+                    <a>Consumers</a> do so by <emph>dereferencing</emph> the target URI,
+                    that is, fetching the resource representation by following a link.
+                </p>
+                <p>
+                    W3C WoT follows the definitions of Web Linking [[!RFC8288]],
+                    where a link is comprised of:
+                    <ul>
+                        <li>a link context,</li>
+                        <li>a relation type,</li>
+                        <li>a link target, and</li>
+                        <li>optionally target attributes.</li>
+                    </ul>
+                </p>
+                <p>
+                    Link relation types are either IANA-registered tokens adhering to the ABNF
+                    <code style="white-space: nowrap;">LOALPHA *( LOALPHA / DIGIT / "." / "-" )</code>
```

Comment:
it would be useful to have examples and link to the  IANA registry for these types.

Reply:
Added